### DISCUSSION TITLE
Panic "nuclei flag redefined: v"

### DISCUSSION THREAD

**Discussion by 0xPugal** [projectdiscovery/nuclei#1011] [General]
litt1eb0y@hacking:~$ nuclei -h
nuclei flag redefined: v
panic: nuclei flag redefined: v

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc000082180, {0x153c668, 0x1df3a8c}, {0x113249e, 0x1}, {0x11457f6, 0x13})
	/usr/local/go/src/flag/flag.go:879 +0x2f4
flag.BoolVar(...)
	/usr/local/go/src/flag/flag.go:638
github.com/projectdiscovery/goflags.(*FlagSet).BoolVarP(0xc0002e5860, 0x1df3a8c, {0x11377e0, 0x1df3978}, {0x113249e, 0x1147e28}, 0x0, {0x11457f6, 0x0})
	/home/litt1eb0y/go/pkg/mod/github.com/projectdiscovery/goflags@v0.0.7/goflags.go:276 +0x85
main.readConfig()
	/home/litt1eb0y/go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.4.3/cmd/nuclei/main.go:75 +0x830
main.main()
	/home/litt1eb0y/go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.4.3/cmd/nuclei/main.go:19 +0x2b

**Comment by ehsandeep** [comment]
@litt1e-b0y you are running an outdated version of nuclei, the latest version is [v2.5.1](https://github.com/projectdiscovery/nuclei/releases/tag/v2.5.1)

**Reply by 0xPugal** [reply]
No It's Not Working :(
```
litt1eb0y@hacking:~$ nuclei -update
nuclei flag redefined: v
panic: nuclei flag redefined: v

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc000080180, 0x169b300, 0x1e9e964, 0x1392838, 0x1, 0x13a6823, 0x13)
	/home/litt1eb0y/.go/src/flag/flag.go:871 +0x485
flag.BoolVar(...)
	/home/litt1eb0y/.go/src/flag/flag.go:636
github.com/projectdiscovery/goflags.(*FlagSet).BoolVarP(0xc0002d5aa0, 0x1e9e964, 0x1397c71, 0x7, 0x1392838, 0x1, 0x0, 0x13a6823, 0x13, 0x31)
	/home/litt1eb0y/go/pkg/mod/github.com/projectdiscovery/goflags@v0.0.7/goflags.go:276 +0x85
main.readConfig()
	/home/litt1eb0y/go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.5.1/cmd/nuclei/main.go:81 +0xc8e
main.main()
	/home/litt1eb0y/go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.5.1/cmd/nuclei/main.go:21 +0x34
```

**Reply by forgedhallpass** [reply]
With the latest versions of nuclei this issue is not reproducible.

![image](https://user-images.githubusercontent.com/13679401/146951265-28afd098-01df-4973-b9c1-6a19ae2aed9f.png)
